Added new-pr-autoresponder and Stale-Bot Functionality (#3)
civicrm/probot-civicrm: config/civicrm/civicrm-core/PR_NEW_REPLY_TEMPLATE.mustache.md

(*Standard links*)

> * __If this is your first pull-request for CiviCRM__, please browse [CONTRIBUTING.md](https://github.com/civicrm/civicrm-core/blob/master/.github/CONTRIBUTING.md) for information about the development and testing processes. Also, please update your details in [contributor-key.yml](https://github.com/civicrm/civicrm-core/blob/master/contributor-key.yml) through a pull request.

Reviewer: ```suggestion
* __If this is your first pull-request for CiviCRM__, please read [CONTRIBUTING.md](https://github.com/civicrm/civicrm-core/blob/master/.github/CONTRIBUTING.md) for information about the development and testing processes. Please add or update your details in [contributor-key.yml](https://github.com/civicrm/civicrm-core/blob/master/contributor-key.yml) through a pull request.
```

Author: I have committed the suggestion 👍
